### PR TITLE
feat: add persistent/url to return

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: main.star [hunk 1]
```diff
@@ -15,23 +18,30 @@ def run(
...
     max_memory=REDIS_MAX_MEMORY,  # type:int
-    node_selectors=None,    #type:dict
+    node_selectors=None,  # type:dict
 ):
     """
     This will return a struct that contains the following properties:
     - service_name: the name of the service that was created
     - hostname: a future reference (https://docs.kurtosis.com/concepts-reference/future-references) to the hostname of the created Redis service
-    - port_number: the port number of the client port
+    - port_number: the port number of the client port\
```
Comment (h4ck3rk3y) [inline main.star:32]: what is that slash doing?

File: main.star [hunk 2]
```diff
@@ -15,23 +18,30 @@ def run(
...
+    - persistent(bool): Whether the data should be persisted. Defaults to True; Note that this isn't supported on multi node k8s cluster as of 2023-10-16
```
Comment (h4ck3rk3y) [inline main.star:33]: but it is now with PV / PVC work?

**Review by h4ck3rk3y — APPROVED** [review]

File: main.star (hunk 2 above)
Comment (barnabasbusa) [reply main.star:33]: ah yeah that should have been fixed now

Comment (barnabasbusa) [reply main.star]: fixed the comment

**Pull request merged**